sweetest/src/main/java/com/mysugr/sweetest/framework/dependency/DependencyManager.kt

```diff
@@ -19,7 +19,7 @@ class DependencyManager(setupHandlerReceiver: (DependencySetupHandler) -> Unit)
 
     private val initializerContext = object : DependencyInitializerContext() {
         override fun <T : Any> instanceOf(clazz: KClass<T>): T {
-            val configuration = configurations.getAssignableTo(clazz)
+            val configuration = requireNotNull(configurations.getAssignableTo(clazz))
```

Comment:
now that `getAssignableTo` doesn't throw an exception with a well explaining error message anymore it should be done here i think